pkg/common/quotas/quotas.go

```diff
@@ -0,0 +1,236 @@
+package quotas
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+
+	corev1 "k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/api/resource"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/selection"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	orgLabelName = "appuio.io/organization"
+
+	// Namespace related quotas
```

Comment:
These values are hardcoded in AppCat but they are managed by APPUiO Cloud. Are there any APIs that we use to get these values?

Reply:
I've talked with Simon and with the current setup there's no sensible way to fetch those values. They are working on improving this though. So for now we have the hardcoded values.